### PR TITLE
Return the node info as a header

### PR DESCRIPTION
**Pull request by Exide**
REST responses and item's delivered via webhooks now include the following header:

    Hub-Node: <domain>:<port>

### CODE REVIEW

**Comment by Exide** [comment]
This is running in pdx.dev currently.

File: src/main/java/com/flightstats/hub/app/HubResponseFilter.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import javax.ws.rs.container.ContainerResponseContext;
+import javax.ws.rs.container.ContainerResponseFilter;
+import javax.ws.rs.ext.Provider;
+
+@Provider
+public class HubResponseFilter implements ContainerResponseFilter {
+
+    @Override
+    public void filter(ContainerRequestContext requestContext, ContainerResponseContext responseContext) {
+        responseContext.getHeaders().add("Hub-Node", HubHost.getLocalNamePort());
```
Comment (crozierm) [inline src/main/java/com/flightstats/hub/app/HubResponseFilter.java]: Curious - how does this get the real name while inside of docker?

**Comment by Paul-Hess** [comment]
This looks great to me

File: src/main/java/com/flightstats/hub/app/HubResponseFilter.java (hunk 1 above)
Comment (Exide) [reply src/main/java/com/flightstats/hub/app/HubResponseFilter.java]: I believe its a byproduct of Docker's `host` network. If you jump in one of the running Hub containers it thinks it's hostname is that of it's host.

Comment (Exide) [reply src/main/java/com/flightstats/hub/app/HubResponseFilter.java]: We're just calling `InetAddress.getLocalHost().getHostName()` in the application code.

Comment (Exide) [reply src/main/java/com/flightstats/hub/app/HubResponseFilter.java]: There is an open issue for when running this in the "single" Hub:
https://github.com/flightstats/hub/issues/1028

**Pull request merged**